tencentcloud/resource_tc_instance.go

```diff
@@ -309,6 +309,13 @@ func resourceTencentCloudInstance() *schema.Resource {
 							ForceNew:    true,
 							Description: "Decides whether the disk is encrypted. Default is `false`.",
 						},
+						"throughput_performance": {
+							Type:        schema.TypeInt,
+							Optional:    true,
+							Default:     0,
+							ForceNew:    true,
+							Description: "This parameter used to make extra performance of the data disk. It's only supported with `CLOUD_TSSD` and `CLOUD_HSSD` disk type.",
```

Comment:
Add extra performance to the data disk. Only works when disk type is `CLOUD_TSSD` or `CLOUD_HSSD`.